ngx-fudis/projects/ngx-fudis/src/lib/components/button/button.component.html

```diff
@@ -1,24 +1,28 @@
 <ng-container *ngIf="tooltip">
   <button
     #buttonElement
+    fudisTooltip
+    [class]="_classList | async"
     [type]="type"
     (blur)="_handleButtonBlur($event)"
     (focus)="_handleButtonFocus($event)"
+    (click)="!disabled && buttonClick($event)"
```

Comment:
!disabled here doesn't have any effect

Reply:
If the button is disabled we don't want click to emit any event. It is necessary attribute.